visualization/dashboard/src/MLIDashboard/Controls/WhatIfTab/WhatIfTab.tsx

```diff
@@ -941,14 +1267,23 @@ export class WhatIfTab extends React.PureComponent<IWhatIfTabProps, IWhatIfTabSt
         this.setState({ selectedFeatureKey: item.key as string });
     }
 
+    private onICEClassSelected(event: React.FormEvent<IComboBox>, item: IDropdownOption): void {
```

Comment:
it seems that IDropdownOption is obsolete?
https://docs.microsoft.com/en-us/javascript/api/office-ui-fabric-react/idropdownoption?view=office-ui-fabric-react-latest
should we be using selected?  Also, any way to make item.key keep the type so it wouldn't have to be casted?

Reply:
IDropdownOption isn't obsolete, the isSelected property on it is, which we don't use (I hope)

Reply:
We could just as well use the base ISelectableObject instead, since we don't use the selected property, no real difference.

Reply:
good point, just the property is obsolete, but then IDropdownOption basically doesn't have anything else on it, so I would think we might as well use ISelectableOption.